var/spack/repos/builtin/packages/py-iminuit/package.py

```diff
@@ -12,6 +12,7 @@ class PyIminuit(PythonPackage):
     homepage = "https://pypi.python.org/pypi/iminuit"
     url      = "https://pypi.io/packages/source/i/iminuit/iminuit-1.2.tar.gz"
 
+    version('1.3.6', sha256='d79a197f305d4708a0e3e52b0a6748c1a6997360d2fbdfd09c022995a6963b5e')
     version('1.2', sha256='7651105fc3f186cfb5742f075ffebcc5088bf7797d8ed124c00977eebe0d1c64')
 
     # Required dependencies
```

Comment:
Numpy isn't optional, it's required. Matplotlib is only needed for testing, along with a few other dependencies, so you can make it `type='test'`. I think the same is true for cython, it's only required for testing and if you're building from the GitHub sources, the PyPI sources should already be cythonized.